tests/test_polys.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+
+import numpy as np
+
+from UncertainSCI.families import JacobiPolynomials
+from UncertainSCI.opolynd import TensorialPolynomials
+from UncertainSCI.indexing import LpSet
+
+class PolysTestCase(unittest.TestCase):
+    """
+    Testing orthogonal polynomial routines.
+    """
+
+    def setUp(self):
+        self.longMessage = True
+
+    def test_derivative_expansion(self):
+        """
+        Expansion coefficients of derivatives.
+        """
+
+        alpha = 10*np.random.rand(1)[0]
+        beta = 10*np.random.rand(1)[0]
+        J = JacobiPolynomials(alpha=alpha, beta=beta)
+
+        N = 13
+        K = 11
+
+        x, w = J.gauss_quadrature(2*N)
+        V = J.eval(x, range(K+1))
+
+        for s in range(4):
+            C = J.derivative_expansion(s, N, K)
+
+            Vd = J.eval(x, range(N+1), d=s)
+            C2 = Vd.T @ np.diag(w) @ V
```

Comment:
Is there any specific reason for the use of @ here instead of dot or numpy dot?

Reply:
Not particularly, mostly for readability. There seems to be a preference for `@` if it's matrix multiplication (which the above is), and `dot` if it's "other kinds" of dot products, but it seems it's just for style.

[https://numpy.org/doc/stable/reference/generated/numpy.dot.html](https://numpy.org/doc/stable/reference/generated/numpy.dot.html)